### ISSUE TITLE
AXEnhancedUserInterface attribute issues discovered using 1Password 8

### ISSUE THREAD

**Issue by tombell** [kasper/phoenix#301]
Since [Stark](https://github.com/starkwm/stark) and Phoenix are quite similar, I thought I'd share something I've come across with Stark since upgrading to 1Password 8 for mac.

I noticed, after using the 1Password 8 extension for Chrome to log into a site, then using my window manager to move/size the window, it acted strangely. I can consistently reproduce this by closing Chrome, filling in a password, and trying again. I decided to download and setup Phoenix with similar key binds and positions/sizes to see if it was just my implementation, however Phoenix appears to suffer the same way.

[This tweet containing a video](https://twitter.com/thetombell/status/1522283863300624384) showing off the weirdness. I've also send an email to 1Password support to see if they can share any light on it.

The 1Password 8 update requires accessibility permissions now, so I suspect they're doing something weird to talk to Chrome, and I'm unsure how to figure out what if anything is going on.

Thought I'd report just incase others start seeing similar behaviour with 1Password 8 for mac being released this week.

**Comment by tombell** [comment]
After some more digging last night, this isn't to do with the Chrome extension as I first thought.

It's due to the `AXEnhancedUserInterface` attribute being set on the application, and causing the repositioning and resizing issues.

I'm working on fixing this in [Stark in this draft MR](https://github.com/starkwm/stark/pull/19), I believe Phoenix will be able to use the similar calls to fix this as well.

**Comment by Kauhsa** [comment]
Yeah, I have a similar issue in Google Chrome in general, and I'm not using the 1Password extension for it. I tried to fix this in #310.
